source/overview/license-and-subscription.rst

```diff
@@ -110,6 +110,45 @@ We use `Stripe <https://stripe.com/payments>`_ and `Solupay <https://www.solupay
 
 Should you wish to make payment using another method, please contact our `Billing team <mailto:AR@mattermost.com>`_.
 
+Why do I need to provide my physical address?
```

Comment:
```suggestion
Why do I need to provide my physical address when purchasing a Self-Managed Enterprise license?
```

Reply:
@jamiehurewitz per our earlier conversation, this applies for self-managed licenses correct? Because for Cloud, we're basing it on the billing/payment address.

https://community.mattermost.com/private-core/pl/duojwkwzc3ds5cys7gr4a5nskh

Reply:
That is not necessarily true. We use the address also for back up research purposes as well. For example, if we get a hit on a Jason Blais, and the customer is located in Canada but the individual that is on the denied party list which caused the hit, is actually located in France, we use the address to help determine whether the hit we got is a legitimate hit and are unable to proceed with the transaction or if it is just someone with the same name, thus a false flag and can clear the transaction to proceed.

Reply:
@jamiehurewitz Makes sense, but we are not collecting the physical address for Cloud transactions though. So I just want to make sure our documentation is accurate.

We could change the title to `Why do I need to provide my name and physical address?` in which case the answer would apply to cloud and self-managed.

Reply:
I will leave it up to you to decide. I just wanted you to understand how the information is used.

Reply:
I've edited your suggestion, @jasonblais - I think it works well based on the thread above.